Typecheck task fails

```bash
> tsc -p tsconfig-typecheck-jsdoc.json

node_modules/@types/parse5/lib/tree-adapters/default.d.ts:1:25 - error TS2792: Cannot find module '../../'. Did you mean to set the 'moduleResolution' option to 'node', or to add aliases to the 'paths' option?

1 import * as parse5 from '../../';
                          ~~~~~~~~

node_modules/@types/trusted-types/index.d.ts:10:22 - error TS2792: Cannot find module './lib'. Did you mean to set the 'moduleResolution' option to 'node', or to add aliases to the 'paths' option?

10 import * as lib from './lib';
                        ~~~~~~~

tsconfig-typecheck-jsdoc.json:11:5 - error TS5023: Unknown compiler option 'exactOptionalPropertyTypes'.

11     "exactOptionalPropertyTypes": false,
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

tsconfig-typecheck-jsdoc.json:20:25 - error TS6046: Argument for '--moduleResolution' option must be: 'node', 'classic'.

20     "moduleResolution": "NodeNext"
                           ~~~~~~~~~~


Found 4 errors.
```

--- a/tsconfig-typecheck-jsdoc.json
+++ b/tsconfig-typecheck-jsdoc.json
@@ -8,7 +8,6 @@
     "allowUnusedLabels": false,
     "alwaysStrict": true,
     "checkJs": true,
-    "exactOptionalPropertyTypes": false,
     "noEmit": true,
     "noFallthroughCasesInSwitch": true,
     "noImplicitAny": true,
@@ -17,6 +16,6 @@
     "noImplicitThis": true,
     "noPropertyAccessFromIndexSignature": false,
     "target": "ESNext",
-    "moduleResolution": "NodeNext"
+    "moduleResolution": "node"
   }
 }

